config/middleware.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    settings: {
+      cors: {
+        enabled: true,
+        origin: ['http://localhost:3000'], // replace with the domain of your Next.js application
```

Comment:
I am merging this pull request for now, but I'll modify this line to include our two domains (athleti.fi and athletifiselect.com) for production